apis/apiextensions/v1/xrd_types.go

```diff
@@ -103,6 +107,23 @@ type CompositionReference struct {
 	Name string `json:"name"`
 }
 
+// CompositeResourceDefinitionSpecMetadata specifies the desired metadata of the defined composite resource.
+type CompositeResourceDefinitionSpecMetadata struct {
+	// Map of string keys and values that can be used to organize and categorize
+	// (scope and select) objects. May match selectors of replication controllers
+	// and services.
```

Comment:
Should we also document the behavior with labels, maybe here? something like "additional labels" for CRDs...

Reply:
I added a proposal.

Reply:
here i'm not sure about the `'`
https://github.com/crossplane/crossplane/pull/4310/files#diff-23609283c12720d62d533c0151d9b3830bb4b255ae2ae77b7586e66f5cf2f8adR270

Reply:
I just checked the applied CRDs in a cluster and it seems to look good for me
```
$ k explain CompositeResourceDefinition.spec.metadata       
GROUP:      apiextensions.crossplane.io
KIND:       CompositeResourceDefinition
VERSION:    v1

FIELD: metadata <Object>

DESCRIPTION:
    Metadata specifies the desired metadata for the defined composite resource
    and claim CRD's.
    
FIELDS:
  annotations   <map[string]string>
    Annotations is an unstructured key value map stored with a resource that may
    be set by external tools to store and retrieve arbitrary metadata. They are
    not queryable and should be preserved when modifying objects. More info:
    https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations

  labels        <map[string]string>
    Map of string keys and values that can be used to organize and categorize
    (scope and select) objects. May match selectors of replication controllers
    More info:
    https://kubernetes.io/docs/concepts/overview/working-with-objects/labels and
    services. These labels are added to the composite resource and claim CRD's
    in addition to any labels defined by `CompositionResourceDefinition`
    `metadata.labels`.

```